content/foundations/color/overview.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: UI color system
+description:
+  Color is a fundamental part of Primer's visual language. In this guide you will learn about the principles,
+  patterns, and abstractions that make up Primer’s color system.
+---
+
+import {Box} from '@primer/react'
+
+## Overview
+
+<img width="960" alt="wip" src="https://github.com/primer/primitives/assets/18661030/615e3e06-9289-4adc-ab85-2213fae96e8c" style="margin-bottom: 1rem;" />
+
+GitHub's UI offers two color modes, `light` and `dark`. Primer supports both color modes across nine [themes](../../primitives/getting-started#theming). Every pattern in Primer is built to work across all color modes out of the box.
+
+### Color modes in Figma
+
+When designing product interfaces in Figma, we recommend using **light mode** or **dark mode** as the Primer Web Figma library provides components and tokens in those two modes.
+You can learn more about using design tokens in Figma in our [Figma guidelines](../../guides/figma/getting-started#styles-and-variables).
+
+### Color design tokens
+
+Primer delivers colors in the form of [design tokens](../../primitives/getting-started). Design tokens are a layer of abstraction that allows for better maintainability, consistency and theming. Within Primer, design tokens are represented as CSS variables for code and Figma variables for design. For a full reference table of all available design tokens, see the [color CSS variables](../../primitives/color) page.
+
+As an example of how color tokens work, when `bgColor-default` is referenced for a background color, the value of that token will automatically change depending on the color mode.
+
+<img
+  width="960"
+  alt="Same component shown in light mode and dark mode color palettes"
+  src="https://github.com/primer/react/assets/18661030/21726ed0-2175-44a3-a821-af002778c0b3"
+/>
+
+### Design token categories
+
+Primer design tokens are categorized into three groups: **base**, **functional**, and **component/pattern**. To read more about the naming convention, see the [design token naming guidelines](../../primitives/naming-convention) page.
```

Comment:
```suggestion
Primer design tokens are categorized into three groups:
- base
- functional
- component/pattern

To read more about the naming convention, see the [design token naming guidelines](../../primitives/naming-convention) page.
```

Could be a list. Comparison:
![Screenshot of section as current and as suggested, a list](https://github.com/primer/design/assets/3369400/2b44d2ae-d4d9-44c4-bc34-eb86d0a93ec5)